pkgs/servers/home-assistant/custom-components/rako/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  rakopy,
+  nix-update-script,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "princekama";
+  domain = "rako";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "princekama";
+    repo = "home-assistant-rako";
+    rev = "v${version}";
+    hash = "sha256-IwtYigUflnXgISrP/+F75Jy7G2WrgIxoyzqG4zoYP34=";
+  };
+
+  dependencies = [ rakopy ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242